### PR TITLE
Relegate some more verbose output as "debug"

### LINKED ISSUE #13: The current build is very noisy
Looks like there's a lot of extra 'print' here and there in the code that wasn't there before the new session-dbus branch went in.  For example, running `socranop-ctl --no-dbus -l` now outputs all this when it should really only be the last 14 lines or so.

We should remove these extra prints or at least hide them behind a `--debug` or `--verbose` flag...

```
AbstractDirs.__init__ UsrLocalDirs(chroot=None, prefix='/usr/local', datadir='/usr/local/share', statedir='/home/jramsay/.config/socranop/state') chroot=None
AbstractDirs.__init__ UsrDirs(chroot=None, prefix='/usr', datadir='/usr/share', statedir='/home/jramsay/.config/socranop/state') chroot=None
AbstractDirs.__init__ HomeDirs(chroot=None, prefix='/home/jramsay/.local', datadir='/home/jramsay/.local/share', statedir='/home/jramsay/.config/socranop/sta
te') chroot=None                                                              
Using dirs: HomeDirs(chroot=None, prefix='/home/jramsay/.local', datadir='/home/jramsay/.local/share', statedir='/home/jramsay/.config/socranop/state')
Using stateDir PosixPath('/home/jramsay/.config/socranop/state')              
Found device DEVICE ID 05fc:0032 on Bus 005 Address 002 =================     
 bLength                :   0x12 (18 bytes)
 bDescriptorType        :    0x1 Device                                       
 bcdUSB                 :  0x200 USB 2.0                                      
 bDeviceClass           :   0xef Miscellaneous
 bDeviceSubClass        :    0x2                                              
 bDeviceProtocol        :    0x1                                              
 bMaxPacketSize0        :   0x40 (64 bytes)
 idVendor               : 0x05fc                                              
 idProduct              : 0x0032                                              
 bcdDevice              :  0x109 Device 1.09
 iManufacturer          :    0x1 Soundcraft                                   
 iProduct               :    0x3 Notepad-12FX
 iSerialNumber          :    0x0                                              
 bNumConfigurations     :    0x2                                              
  CONFIGURATION 1: 500 mA ==================================                                                                                                 
   bLength              :    0x9 (9 bytes)                  
   bDescriptorType      :    0x2 Configuration
   wTotalLength         :  0x129 (297 bytes)
   bNumInterfaces       :    0x4    
   bConfigurationValue  :    0x1                                              
   iConfiguration       :    0x0                                              
   bmAttributes         :   0x80 Bus Powered
   bMaxPower            :   0xfa (500 mA)                                     
    INTERFACE 0: Audio =====================================                  
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface                  
     bInterfaceNumber   :    0x0                                              
     bAlternateSetting  :    0x0                                              
     bNumEndpoints      :    0x0
     bInterfaceClass    :    0x1 Audio                                        
     bInterfaceSubClass :    0x1                                              
     bInterfaceProtocol :   0x20                                              
     iInterface         :    0x3 Notepad-12FX
    INTERFACE 1: Audio =====================================                                                                                                 
     bLength            :    0x9 (9 bytes)                                                                                                                   
     bDescriptorType    :    0x4 Interface                                                                                                                   
     bInterfaceNumber   :    0x1                                                                                                                             
     bAlternateSetting  :    0x0                                                                                                                             
     bNumEndpoints      :    0x0                                                                                                                             
     bInterfaceClass    :    0x1 Audio                                                                                                                       
     bInterfaceSubClass :    0x2                                                                                                                             
     bInterfaceProtocol :   0x20                                                                                                                             
     iInterface         :    0x4 Notepad                                                                                                                     
    INTERFACE 1, 1: Audio ==================================
     bLength            :    0x9 (9 bytes)                                                                                                                   
     bDescriptorType    :    0x4 Interface                                                                                                                   
     bInterfaceNumber   :    0x1                                              
     bAlternateSetting  :    0x1                                                                                                                             
     bNumEndpoints      :    0x2                                                                                                                             
     bInterfaceClass    :    0x1 Audio                                                                                                                       
     bInterfaceSubClass :    0x2                                              
     bInterfaceProtocol :   0x20                                                                                                                             
     iInterface         :    0x4 Notepad                                      
      ENDPOINT 0x1: Isochronous OUT ========================                                                                                                 
       bLength          :    0x7 (7 bytes)                               
       bDescriptorType  :    0x5 Endpoint                                
       bEndpointAddress :    0x1 OUT                                          
       bmAttributes     :    0x5 Isochronous
       wMaxPacketSize   :   0x70 (112 bytes)                                  
       bInterval        :    0x1                                              
      ENDPOINT 0x81: Isochronous IN ========================                                                                                       [157/1902]
       bLength          :    0x7 (7 bytes)                                                                                                                   
       bDescriptorType  :    0x5 Endpoint                                                                                                                    
       bEndpointAddress :   0x81 IN                                                                                                                          
       bmAttributes     :   0x11 Isochronous                                                                                                                 
       wMaxPacketSize   :    0x4 (4 bytes)                                    
       bInterval        :    0x4                                              
    INTERFACE 1, 2: Audio ==================================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x1
     bAlternateSetting  :    0x2
     bNumEndpoints      :    0x2
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x2
     bInterfaceProtocol :   0x20
     iInterface         :    0x4 Notepad 
      ENDPOINT 0x1: Isochronous OUT ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :    0x1 OUT
       bmAttributes     :    0x5 Isochronous
       wMaxPacketSize   :   0x38 (56 bytes)
       bInterval        :    0x1
      ENDPOINT 0x81: Isochronous IN ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :   0x81 IN
       bmAttributes     :   0x11 Isochronous
       wMaxPacketSize   :    0x4 (4 bytes)
       bInterval        :    0x4
    INTERFACE 2: Audio =====================================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x2
     bAlternateSetting  :    0x0
     bNumEndpoints      :    0x0
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x2
     bInterfaceProtocol :   0x20
     iInterface         :    0x5 Notepad 
    INTERFACE 2, 1: Audio ==================================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x2
     bAlternateSetting  :    0x1
     bNumEndpoints      :    0x1
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x2
     bInterfaceProtocol :   0x20
     iInterface         :    0x5 Notepad 
      ENDPOINT 0x82: Isochronous IN ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :   0x82 IN
       bmAttributes     :    0x5 Isochronous
       wMaxPacketSize   :   0x70 (112 bytes)
       bInterval        :    0x1
    INTERFACE 3: Application Specific ======================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x3
     bAlternateSetting  :    0x0
     bNumEndpoints      :    0x0
     bInterfaceClass    :   0xfe Application Specific
     bInterfaceSubClass :    0x1
     bInterfaceProtocol :    0x1
     iInterface         :    0xa Soundcraft DFU
  CONFIGURATION 1: 500 mA ==================================
   bLength              :    0x9 (9 bytes)
   bDescriptorType      :    0x2 Configuration
   wTotalLength         :  0x129 (297 bytes)
   bNumInterfaces       :    0x4
   bConfigurationValue  :    0x1
   iConfiguration       :    0x0 
   bmAttributes         :   0x80 Bus Powered
   bMaxPower            :   0xfa (500 mA)
    INTERFACE 0: Audio =====================================                                                                                        [80/1902]
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x0
     bAlternateSetting  :    0x0
     bNumEndpoints      :    0x0
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x1
     bInterfaceProtocol :   0x20
     iInterface         :    0x3 Notepad-12FX
    INTERFACE 1: Audio =====================================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x1
     bAlternateSetting  :    0x0
     bNumEndpoints      :    0x0
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x2
     bInterfaceProtocol :   0x20
     iInterface         :    0x4 Notepad 
    INTERFACE 1, 1: Audio ==================================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x1
     bAlternateSetting  :    0x1
     bNumEndpoints      :    0x2
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x2
     bInterfaceProtocol :   0x20
     iInterface         :    0x4 Notepad 
      ENDPOINT 0x1: Isochronous OUT ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :    0x1 OUT
       bmAttributes     :    0x5 Isochronous
       wMaxPacketSize   :   0x70 (112 bytes)
       bInterval        :    0x1
      ENDPOINT 0x81: Isochronous IN ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :   0x81 IN
       bmAttributes     :   0x11 Isochronous
       wMaxPacketSize   :    0x4 (4 bytes)
       bInterval        :    0x4
    INTERFACE 1, 2: Audio ==================================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x1
     bAlternateSetting  :    0x2
     bNumEndpoints      :    0x2
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x2
     bInterfaceProtocol :   0x20
     iInterface         :    0x4 Notepad 
      ENDPOINT 0x1: Isochronous OUT ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :    0x1 OUT
       bmAttributes     :    0x5 Isochronous
       wMaxPacketSize   :   0x38 (56 bytes)
       bInterval        :    0x1
      ENDPOINT 0x81: Isochronous IN ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :   0x81 IN
       bmAttributes     :   0x11 Isochronous
       wMaxPacketSize   :    0x4 (4 bytes)
       bInterval        :    0x4
    INTERFACE 2, 1: Audio ==================================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x2
     bAlternateSetting  :    0x1
     bNumEndpoints      :    0x1
     bInterfaceClass    :    0x1 Audio
     bInterfaceSubClass :    0x2
     bInterfaceProtocol :   0x20
     iInterface         :    0x5 Notepad 
      ENDPOINT 0x82: Isochronous IN ========================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :   0x82 IN
       bmAttributes     :    0x5 Isochronous
       wMaxPacketSize   :   0x70 (112 bytes)
       bInterval        :    0x1
    INTERFACE 3: Application Specific ======================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x3
     bAlternateSetting  :    0x0
     bNumEndpoints      :    0x0
     bInterfaceClass    :   0xfe Application Specific
     bInterfaceSubClass :    0x1
     bInterfaceProtocol :    0x1
     iInterface         :    0xa Soundcraft DFU
<socranop.notepad.Notepad_12fx object at 0x7fb7e9804be0> using existing stateFile /home/jramsay/.config/socranop/state/Notepad-12FX.state
Detected a Notepad-12FX (fw v1.09)
-----------------------------
capture_1 <- Mic/Line 1
capture_2 <- Mic/Line 2
-----------------------------
             Mic/Line 3   [0]
             Mic/Line 4   
capture_3 <- Stereo 5/6 L [1]
capture_4 <- Stereo 5/6 R 
             Stereo 7/8 L [2]
             Stereo 7/8 R 
             Mix L        [3]
             Mix R        
-----------------------------
```

### PR DESCRIPTION
**Pull request by lack**
Fixes #13

### CODE REVIEW

**Review by ndim — COMMENTED** [review]
No --verbose for the dbus service?

**Comment by lack** [comment]
Good point :)

**Comment by lack** [comment]
I added a --verbose flag to all the top-level entry points, even `socranop-installtool` but for now all the installation print lines are untouched.

File: socranop/common.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020,2021 Jim Ramsay <i.am@jimramsay.com>
+# Copyright (c) 2020,2021 Hans Ulrich Niedermann <hun@n-dimensional.de>
```
[the remark this answers is not in the document]
Comment (ndim) [reply socranop/common.py:3]: OK, with the --version code I have.

**Review by ndim — APPROVED** [review]
Looks good.

**Pull request merged**